app/helpers/SessionAnswersHelper.scala

```diff
@@ -473,4 +474,19 @@ class SessionAnswersHelper @Inject()(uploadJourneyRepository: UploadJourneyRepos
     userRequest.answers.getAnswer[String](SessionKeys.doYouWantToAppealBothPenalties).contains("no") &&
       userRequest.answers.getAnswer[LocalDate](SessionKeys.dateCommunicationSent).exists(_.isBefore(dateTimeNow.minusDays(appConfig.daysRequiredForLateAppeal)))
   }
+
+  def viewAppealRows()(implicit userRequest: UserRequest[_], messages: Messages): Seq[CheckYourAnswersRow] = {
+     Seq(CheckYourAnswersRow(
+       messages("viewAppealDetails.vrn"),
+       userRequest.vrn,
+       ""
+     ),
+    CheckYourAnswersRow(
+      messages("viewAppealDetails.penaltyAppealed"),
+      (PenaltyTypeHelper.getKeysFromSession().get.head + ": " +
+        PenaltyTypeHelper.getKeysFromSession().get(1) + " to " +
```

Comment:
Having 'to' means that this hard-coded test will not be translated

Reply:
Also the penalty period may not be in the session, which I suspect is what this is doing?

Reply:
_text_ not test

> Having 'to' means that this hard-coded test will not be translated